server/server.go

```diff
@@ -1695,7 +1718,8 @@ func (s *Server) createClient(conn net.Conn) *client {
 	}
 	now := time.Now()
 
-	c := &client{srv: s, nc: conn, opts: defaultOpts, mpay: maxPay, msubs: maxSubs, start: now, last: now}
+	c := &client{srv: s, nc: conn, opts: defaultOpts, mpay: maxPay, msubs: maxSubs, start: now, last: now, flags: initalFlags}
+	ws := c.flags.isSet(wsClient)
```

Comment:
Might be nice to use this to log that it is a `ws` connection when logging in debug level below...

```
cid:1 - Client websocket connection created
```

Reply:
Or we could also do `wid` or similar in initClient(). Wasn't sure if I should do that or not.

Reply:
I like `wid`, and would also follow similar convention as with rid, gid for example

Reply:
Done